### PR TITLE
Fixes #11885: Autocompletion for rudder commands doesn't work on RHEL and SLES

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/11885

### CODE REVIEW

**Comment by amousset** [comment]
Missing change in Makefile

**Comment by peckpeck** [comment]
Commit modified

File: rudder-agent/SOURCES/Makefile [hunk 1]
```diff
@@ -194,7 +194,7 @@ endif
 
         # Install /etc/bash_completion.d/rudder in order to autocomplete rudder commands
```
Comment (amousset) [inline rudder-agent/SOURCES/Makefile]: rudder.sh

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**